packages/x-components/src/components/base-teleport.vue

```diff
@@ -13,7 +13,7 @@ export default defineComponent({
   props: {
     /** The element or css selector to which the component will be teleported. */
     target: {
-      type: [String, Element] as PropType<string | Element>,
+      type: Object as PropType<string | Element>,
```

Comment:
```suggestion
      type:[String, Object] as PropType<string | Element>,
```
String is also allowed